aws_run_celery.py

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env python
+from flask import Flask
+
 from app import notify_celery, create_app  # noqa: notify_celery required to get celery running
 from credstash import getAllSecrets
 import os
 
 os.environ['AWS_REGION'] = 'eu-west-1'
 os.environ.update(getAllSecrets(region="eu-west-1"))
 
-application = create_app()
+application = Flask('delivery')
```

Comment:
does 'delivery' get used or printed anywhere? Maybe ftp-delivery?

Reply:
`delivery` goes into kibana as the `application` type. That said I don't know where the ftp logs have got to - i can't see any on kibana 😓